packages/esm-patient-chart-app/src/patient-chart/chart-review/dashboard-view.scss

```diff
@@ -9,13 +9,31 @@
 
 .dashboard {
   display: grid;
-  grid-template-columns: 1fr 1fr;
+  grid-template-columns: 1fr;
   grid-auto-rows: auto;
-  grid-auto-columns: auto;
   grid-gap: 25px;
   margin: 1.3125rem;
 }
 
+.dashboardContainer {
+  container-name: dashboard;
+  container-type: inline-size;
+}
+
+@container dashboard (width > 1100px){
```

Comment:
The reason to use different values is that it only considers the area that loads the widgets, so it should collapse when the workspace is opened and there's not enough space. Are there any better ways to define these values?

![Kapture 2024-07-15 at 12 53 08](https://github.com/user-attachments/assets/b1246c1a-06d7-4d17-9eed-0d54a8dc431d)

Reply:
Oh, okay! I would just add a comment above that line that explains the reasoning for the value, so that future devs don't go "where on earth did he get that number???"

Reply:
I completely agree with you.